Sources/TokamakStaticHTML/Tokens/Tokens.swift

```diff
@@ -16,8 +16,13 @@ import TokamakCore
 
 extension Color {
   func cssValue(_ environment: EnvironmentValues) -> String {
-    let rgba = _ColorProxy(self).resolve(in: environment)
-    return "rgba(\(rgba.red * 255), \(rgba.green * 255), \(rgba.blue * 255), \(rgba.opacity))"
+    _ColorProxy(self).resolve(in: environment).cssValue
+  }
+}
+
+extension AnyColorBox.ResolvedValue {
+  var cssValue: String {
+    "rgba(\(red * 255), \(green * 255), \(blue * 255), \(opacity))"
   }
```

Comment:
While we're at it, can TokamakCore/Tokens/Color.swift` file be split into multiple files to fix the file length warning?